### DISCUSSION TITLE
No peers - Crusader Kings 3

### DISCUSSION THREAD

**Discussion by idkwhtusrnmtopck** [jc141x/jc141-bash#32] [General]
hi, im trying to download crusader kings 3 from your 1337x account but there are no peers. pls help

**Comment by GrootBlouNaai** [comment]
If you require support, kindly enquire through our matrix rooms.   All releases are seeded 24/7 by a dedicated seedbox. 

Please note we have switch to gitlab as our platform and is dedicated to the project. Discussions are to preferable to be contribution or a suggestion to our project or discussion around our processes. 

Thank you
